makefiles/boot/riotboot.mk

```diff
@@ -19,16 +22,16 @@ EPOCH := $(shell date +%s)
 APP_VER ?= $(EPOCH)
 
 # Final target for slot 0 with riot_hdr
-SLOT0_RIOT_BIN = $(BINDIR_APP)-slot0.$(APP_VER).riot.bin
-SLOT1_RIOT_BIN = $(BINDIR_APP)-slot1.$(APP_VER).riot.bin
+SLOT0_RIOT_BIN = $(BINDIR_RIOTBOOT)/slot0.$(APP_VER).bin
+SLOT1_RIOT_BIN = $(BINDIR_RIOTBOOT)/slot1.$(APP_VER).bin
 SLOT_RIOT_BINS = $(SLOT0_RIOT_BIN) $(SLOT1_RIOT_BIN)
 
 # if RIOTBOOT_SKIP_COMPILE is set to 1, "make riotboot/slot[01](-flash)"
 # will not depend on the base elf files, thus skipping the compilation step.
 # This results in the equivalent to "make flash-only" for
 # "make riotboot/flash-slot[01]".
 ifneq (1, $(RIOTBOOT_SKIP_COMPILE))
-$(BINDIR_APP)-%.elf: $(BASELIBS) $(ARCHIVES)
```

Comment:
@kaspar030 pointed out that this change would have manifests from different apps override each other, so I fixed the issue for different payloads, and bodged it for different applications...

Reply:
No longer a concern, e.g. update files published for different apps go into different directories.

```
→ ls coaproot/fw/
gnrc_networking  suit_update
```